pkg/splunk/enterprise/configuration_test.go

```diff
@@ -236,3 +236,229 @@ func TestSetVolumeDefault(t *testing.T) {
 		}
 	}
 }
+
+func TestSmartstoreApplyIndexerClusterFailsOnInvalidSmartStoreConfi(t *testing.T) {
+	cr := enterprisev1.IndexerCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "standalone",
```

Comment:
Do we want to change the name from "standalone" to something else, since this is not exactly testing smartstore on standalone instance, right?

Reply:
c&p error. Let me fix that.